### PR TITLE
Merging offsets branch into heintz2020

### PR DESCRIPTION
**Pull request by keheintz**
As titled.

### CODE REVIEW

File: frb/builds/build_frbs.py [hunk 1]
```diff
@@ -258,6 +260,8 @@ def frb_190611():
     frb190611.width_err = 0.02 * units.ms
     frb190611.tau = 0.18 * units.ms
     frb190611.tau_err = 0.02 * units.ms
+    frb190611.fluence = 10 * units.Jy * units.ms
+    frb190611.fluence = 2 * units.Jy * units.ms
```
Comment (profxj) [inline frb/builds/build_frbs.py:264]: do you mean `fluence_err`?

File: frb/builds/build_frbs.py [hunk 2]
```diff
@@ -314,6 +318,8 @@ def frb_190711():
     # RM -- Day+2020
     frb190711.RM = 9 * units.rad / units.m**2
     frb190711.RM_err = 2 * units.rad / units.m**2
+    frb190711.fluence = 34 * units.Jy * units.ms
+    frb190711.fluence = 3 * units.Jy * units.ms
```
Comment (profxj) [inline frb/builds/build_frbs.py:322]: same as above

File: frb/builds/build_hosts.py [hunk 3]
```diff
@@ -182,11 +198,12 @@ def build_host_121102(build_photom=False, build_cigale=False, use_orig=False):
     if use_orig:
         host121102.derived['M_r'] = -17.0  # AB; Tendulkar+17
         host121102.derived['M_r_err'] = 0.2  # Estimated by JXP
-        host121102.derived['SFR_nebular'] = 0.23  # MSun/yr; Tendulkar+17
+        host121102.derived['SFR_nebular'] = 0.15  # MSun/yr from new dust-corrected H-alpha; (0.23 from Tendulkar+17)
```
Comment (profxj) [inline frb/builds/build_hosts.py:201]: `use_orig=False`, i.e. this is not called.
Is that what you expected?

File: frb/builds/build_hosts.py [hunk 4]
```diff
@@ -182,11 +198,12 @@ def build_host_121102(build_photom=False, build_cigale=False, use_orig=False):
...
         host121102.derived['Mstar'] = 5.5e7  # Msun; Tendulkar+17
         host121102.derived['Mstar_err'] = 1.5e7  # Msun; Tendulkar+17
     host121102.derived['Z_spec'] = -0.16  # Tendulkar+17 on a scale with Solar O/H = 8.86
     host121102.derived['Z_spec_err'] = -999.  # Tendulkar+17
+    #host121102.derived['logOH'] = 8.69 # Test
```
Comment (profxj) [inline frb/builds/build_hosts.py:206]: remove

File: frb/data/Galaxies/180924/FRB180924_host.json [hunk 5]
```diff
@@ -71,10 +80,6 @@
         "DES_z": 19.832713667943604,
         "DES_z_err": 0.020038900000000002,
         "EBV": 0.0166,
-        "VLT_FORS2_I": 20.071526286300077,
```
Comment (profxj) [inline frb/data/Galaxies/180924/FRB180924_host.json:74]: These should not have been removed.  
You should re-run 180924 and make sure
they reappear.

File: frb/galaxies/offsets.py [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+def angular_offset(frb, galaxy, nsigma=5., nsamp=2000):
+    """
+
+    Warning: All calculations in arcsec -- Do not use for *large* localization error
+
+    Args:
+        frb (frb.frb.FRB):
+        galaxy (frb.galaxies.FRBGalaxy):
+
```
Comment (profxj) [inline frb/galaxies/offsets.py:17]: add `nsigma` and `nsamp` args

**Review by profxj — CHANGES_REQUESTED** [review]
A few modifications to be made.
Thanks

File: frb/builds/build_hosts.py (hunk 3 above)
Comment (keheintz) [reply frb/builds/build_hosts.py:201]: Yes, have included the SFR measurements from the H-alpha line flux further down directly so this should not be called. Have just added the original value now.

File: frb/data/Galaxies/180924/FRB180924_host.json (hunk 5 above)
Comment (keheintz) [reply frb/data/Galaxies/180924/FRB180924_host.json:74]: Done, they now appear.

**Review by profxj — APPROVED** [review]
merging

**Pull request merged**